lametro/models.py

```diff
@@ -664,6 +664,14 @@ def ecomment_message(self):
         else:
             return self.UPCOMING_ECOMMENT_MESSAGE
 
+    @property
+    def todays_meetings(cls):
+        now = app_timezone.localize(datetime.now())
```

Comment:
Academic Q: Django has [a `timezone` util](https://docs.djangoproject.com/en/3.2/topics/i18n/timezones/). Does `timezone.now()` yield the same result as `app_timezone.localize(datetime.now())`?

Reply:
Before reading the docs, I thought about this and was thinking that it would be a big waste for Django to have a `timezone` util if it didn't make use of the ability to set the timezone in the project's `settings.py` file. If it does make use of this setting (which I was assuming the util did), then the two functions should return the same value.

After taking a glance at the docs you linked to and checking `councilmatic/settings.py`, I was able to verify that we do set the timezone to LA time, so they are equivalent here. It's also easier to read, so I think I'll make use of `timezone.now()`.